.buildkite/pipelines/chromium_linux_build/build_chromium.yml

```diff
@@ -0,0 +1,36 @@
+steps:
+  - command: |
+      GITHUB_PR_NUMBER=$GITHUB_ISSUE_NUMBER ts-node .buildkite/scripts/lifecycle/comment_on_pr.ts \
```

Comment:
the `comment-on-pr` script expects the `GITHUB_PR_NUMBER` env to be defined, at the same time passing an issue number should ideally work for writing a comment according to the octokit docs